Rust/sending-transactions.md

```diff
@@ -61,32 +63,33 @@ use web3::types::{TransactionRequest, U256};
 
 ### Connecting to the Node
 
+Add the lines below inside the `fn main()` function:
```

Comment:
I'm not sure about this part because it doesn't always have to be in the main function, it just happens to be there in the example code because its easier.

Reply:
Decided to remove this line for now